### PR TITLE
[15.0][FIX] sale_order_line_date: Same operation as v13 and v14

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**

### CODE REVIEW

**Review by JoanMForgeFlow — APPROVED** [review]
LG!

File: sale_order_line_date/models/sale_order.py [hunk 1]
```diff
@@ -15,27 +15,15 @@ class SaleOrder(models.Model):
...
+        Override method because it destroys the correct operation.
         """
-        res = super(SaleOrder, self)._onchange_expected_date()
-        dates_list = []
-        for line in self.order_line.filtered(
-            lambda x: x.state != "cancel"
-            and not x._is_delivery()
-            and not x.display_type
-        ):
-            if line.commitment_date:
```
Comment (LoisRForgeFlow) [inline sale_order_line_date/models/sale_order.py]: Odoo fixed this themselves :tada: https://github.com/odoo/odoo/commit/449bf686418425a6e4d6c0df68c09f6d77177b02

You can remove it completely

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by rousseldenis** [comment]
PR for v14: https://github.com/OCA/sale-workflow/pull/1902

**Review by rousseldenis — APPROVED** [review]
@BernatPForgeFlow Thanks for forwardports.
Don't know if you know oca-port-pr tool. Can be useful for this kind of task.

https://github.com/OCA/maintainer-tools/pull/504

**Review by HviorForgeFlow — APPROVED** [review]
:+1: @rousseldenis please merge it, `sale_order_line_date` module is broken on all runboat v15 instances forcing to uninstall it, because sale app cannot be opened

Thanks!

**Pull request merged**